ods_ci/tests/Resources/Common.robot

```diff
@@ -477,7 +477,7 @@ Run And Watch Command
 Check Process Output and Status
   [Documentation]    Helper keyward for 'Run And Watch Command', to tail proccess and check its status
   [Arguments]    ${process_id}    ${process_log}    ${temp_log}
-  Log To Console    .    no_newline=true
+  # Log To Console    .    no_newline=true
```

Comment:
If we don't want this log to console maybe we can delete it

Reply:
true, missed it. ty